src/service/cid-api.js

```diff
@@ -92,9 +92,20 @@ export class GoogleCidApi {
       if (!token || this.isStatusToken_(token)) {
         this.persistToken_(TokenStatus.RETRIEVING, TIMEOUT);
       }
+
       const url = GOOGLE_API_URL + apiKey;
       return this.fetchCid_(dev().assertString(url), scope, token)
-          .then(this.handleResponse_.bind(this))
+          .then(response => {
+            const cid = this.handleResponse_(response);
+            if (!cid && response['alternateUrl']) {
```

Comment:
Just checking: The camel case is what the API returns?

Reply:
yes